aries-site/src/examples/templates/Card/Card.js

```diff
@@ -25,13 +37,17 @@ const DesignSystemCard = ({
   icon,
   level,
   media,
+  skeleton,
   subtitle,
   pretitle,
   title,
   ...rest
 }) => {
   const theme = useContext(ThemeContext);
 
+  // demo purposes for using skeleton
+  const skeletonAlign = skeleton ? 'none' : 'start';
```

Comment:
would ne nice to figure out how to avoid having to specify this and still get the right 100% behavior

Reply:
agree I think I would have to go back and look at grommet code